packages/scripts/src/generateIndex.ts

```diff
@@ -129,7 +129,10 @@ export function visitNodes(item: ApiItem, tag: string) {
 
 		members.push({
 			id: idx++,
-			name: member.getScopedNameWithinPackage(),
+			name: (inherited
+				? member.getScopedNameWithinPackage().replace(new RegExp(`^${member.parent?.displayName}`), item.displayName)
```

Comment:
edge case, but this regex will match a class named undefined if no parent...I think checking inherited is supposed to be our guarantee this doesn't happen right?. You could also just check both